idea-plugin/src/main/kotlin/org/jetbrains/compose/desktop/idea/preview/PreviewEntryPoint.kt

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Guess here Android copyright is not applicable.

Reply:
I'm not sure.
Here's the quote from Apache 2.0 license (part 4):
```
...
b. You must cause any modified files to carry prominent notices stating that You changed the files; and
c. You must retain, in the Source form of any Derivative Works that You distribute, all copyright, patent, trademark, and attribution notices from the Source form of the Work, excluding those notices that do not pertain to any part of the Derivative Works; and
...
```
So it seems, that we must both explicitly mention the files are modified and keep the original copyright notices for the derivative parts (but we can state copyright on the modifications).